OpenEdXMobile/src/main/java/org/edx/mobile/view/MyCoursesListFragment.kt

```diff
@@ -390,4 +390,8 @@ class MyCoursesListFragment : OfflineSupportBaseFragment(), RefreshListener {
     override fun isShowingFullScreenError(): Boolean {
         return errorNotification.isShowing
     }
+
+    private fun showPurchaseSuccessSnackbar() {
+        SnackbarErrorNotification(binding.root).showError(R.string.purchase_success_message)
+    }
```

Comment:
method is not getting called better to remove it.

Reply:
removed